android/app/src/main/kotlin/org/getlantern/lantern/service/LanternService.kt

```diff
@@ -42,12 +42,7 @@ open class LanternService : Service(), Runnable {
 
     override fun onCreate() {
         super.onCreate()
-        val serviceIcon: Int = if (LanternApp.session.chatEnabled()) {
-                    R.drawable.status_chat
-                } else {
-                    R.drawable.status_plain
-                }
-        helper = ServiceHelper(this, serviceIcon, R.string.ready_to_connect)
+        helper = ServiceHelper(this, null, R.string.ready_to_connect)
```

Comment:
I already have fixed this issue on my previous PR, If null it will assign a value to serviceHelper class.

Reply:
Sweet! Thanks for making this change. If you make defaultIcon optional with a default value of null, then you can just write

```kotlin
helper = ServiceHelper(this, R.string.ready_to_connect)
```

Reply:
Done